cumulusci/robotframework/Salesforce.py

```diff
@@ -97,6 +97,22 @@ def initialize_location_strategies(self):
         # libraries.
         locator_manager.add_location_strategies()
 
+    def _jsclick(self, locator):
+        """Use javascript to click an element on the page
+
+        See https://help.salesforce.com/articleView?id=000352057&language=en_US&mode=1&type=1
+        """
+
+        self.selenium.wait_until_page_contains_element(locator)
+        element = self.selenium.get_webelement(locator)
+        # Setting the focus first seems to be required as of Spring'20
+        # (read: without it, tests started failing in that release). I
+        # suspect it's because there is a focusOut handler on form
+        # fields which need to be triggered for data to be accepted.
+        self.selenium.driver.execute_script(
+            "arguments[0].focus(); arguments[0].click()", element
+        )
```

Comment:
Did you consider monkeypatching selenium library's click command to do this? Or are there still cases where we prefer to do a standard click?

Reply:
I considered it, but it seemed a bit risky to make the change everywhere. I'm open to the idea, but I'm not a fan of monkey-patching selenium.

Reply:
@boakley yeah, that's fair, especially since this does more than just clicking. It seems like ultimately the ability to interact with shadow DOM elements without resorting to custom javascript is something that belongs in webdriver/selenium, though -- but maybe LWC's efforts to isolate components get in the way.